packages/pipeline-services/src/migration/migrateV2/index.spec.ts

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2018-2021 Elyra Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import migrate from "./";
+
+describe("migrate v1 to v2", () => {
+  it("should change all node paths to relative", () => {
+    const v1 = {
+      pipelines: [
+        {
+          app_data: {
+            name: "name",
+            version: 1,
+          },
+          nodes: [
+            { app_data: { filename: "/user/niko/project/notebook.ipynb" } },
```

Comment:
Should this path be changed?

Reply:
Doesn't really matter, it's just an example path. We can change it to something more random to prevent confusion though. Any suggestions?

Reply:
Hm yeah it's definitely not a big deal although might be better not to use an actual username? It could just be `user/project/notebook.ipynb`